build.gradle

```diff
@@ -68,6 +67,7 @@ allprojects {
             options.errorprone {
                 option('NullAway:AnnotatedPackages', 'com.palantir')
                 option('NullAway:CheckOptionalEmptiness', 'true')
+                option('AllSuggestionsAsWarnings') // https://github.com/google/error-prone/pull/3301
```

Comment:
Very nifty!

Reply:
Hello, the author of the linked Error Prone ticket here!

I just wanted to point out that we maintain our own [fork](https://github.com/PicnicSupermarket/error-prone) of Error Prone where we already support this feature. 
It's a fork that contains a minimal set of changes on top of mainline Error Prone. We've been using it internally for years.
Just to give you an idea; see the extra commits we have on top of the original `v2.15.0` tag here https://github.com/PicnicSupermarket/error-prone/compare/v2.15.0...v2.15.0-picnic-3.